assets/package.json

```diff
@@ -50,7 +50,7 @@
     "svgo-loader": "^2.2.0",
     "ts-jest": "^24.0.0",
     "ts-loader": "^5.3.3",
-    "tslint": "^5.14.0",
+    "tslint": "^5.18.0",
```

Comment:
just curious, is this just keeping things up to date or was there something in `18` that you needed?

Reply:
At some point I was trying to figure out why tslint was using alphabetical ordering instead of "match declaration order". The answer turned out to be what you commented on below, but in the course of getting there I updated this at some point to see if that would affect anything (it didn't) and just left the change in.